app/controllers/items_controller.rb

```diff
@@ -37,7 +37,7 @@ def index
   end
 
   def create
-    create = ItemCreateService.new(organization_id: current_organization.id, item_params: item_params)
+    create = ItemCreateService.new(organization_id: current_organization.id, item_params: item_params, request_unit_ids:)
```

Comment:
Let's protect everything we can with Flipper

```suggestion
    if Flipper.enabled?(:enable_packs)
      create = ItemCreateService.new(organization_id: current_organization.id, item_params: item_params, request_unit_ids:)
    else
      create = ItemCreateService.new(organization_id: current_organization.id, item_params: item_params)
    end
```